src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -15,4 +15,5 @@
     public static final Prefix PREFIX_ADD_NEW_ASSIGNMENT = new Prefix("a/");
     public static final Prefix PREFIX_PERCENTAGE_ASSIGNMENT = new Prefix("%/");
     public static final Prefix PREFIX_RESULT_ASSIGNMENT = new Prefix("r/");
+    public static final Prefix PREFIX_GRADE = new Prefix("ag/");
```

Comment:
I think the prefix should just be called g/, as Im assuming that we add the "addgrade" command in front, so no need to specify the "a" in the prefix.

Reply:
Ok, sounds good.